test/utils.py

```diff
@@ -82,6 +83,22 @@ def wait_file(dir_name, file_pattern, exclude_list, timeout_sec=1):
     return ""
 
 
+def kill_instances():
+    # Kill tarantool instances that were started as children from current process.
+    children = psutil.Process().children(recursive=True)
+    for child in children:
+        os.kill(int(child.pid), 0)
```

Comment:
Do you have it added to raise `ProcessLookupError` if the process does not exists? I don't understand this logic. You already perform the `pid_exists()` check below (it is just wrapper around sending a zero signal, at least on Unix).

Reply:
Ouch, here I wanted to use not a zero signal, but `signal.SIGTERM`

Reply:
Now I see the logic, thanks.